src/navigation/components/Header/Header.tsx

```diff
@@ -114,8 +117,21 @@ export const Header: React.FC<HeaderProps> = ({data, logo}) => {
         setIsSidebarOpened(false);
     }, []);
 
+    useEffect(() => {
+        const handleScroll = () => {
+            if (window.scrollY > 0 && !withBorder) {
+                setWithHeaderBorder(true);
+            } else if (window.scrollY === 0 && !withBorder) {
+                setWithHeaderBorder(false);
+            }
+        };
+
+        window.addEventListener('scroll', _.debounce(handleScroll, 5), {passive: true});
```

Comment:
mb set time to 100?

Reply:
if I set 100 there is big delay on scroll before border shows up. I set 20